### PR TITLE
Update to 7.4.3

### PR DESCRIPTION
**Pull request by p1u3o**

### CODE REVIEW

**Comment by Lctrs** [comment]
Don't forget to update the README

**Comment by p1u3o** [comment]
> Don't forget to update the README

Whoops, fixed.

**Comment by p1u3o** [comment]
@Lctrs Everything look good to you?

**Comment by Lctrs** [comment]
Composer 1.10.0 is out so we might want to update to it. Otherwise, LGTM.

**Comment by p1u3o** [comment]
Alright. I'll merge this when the tests pass.

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ Example Visual Studio Code Configuration
 
 Includes
 
-* [php](https://php.net/) (7.4.2)
-* [composer](https://github.com/composer/composer) (1.9.2)
+* [php](https://php.net/) (7.4.3)
+* [composer](https://github.com/composer/composer) (1.10)
```
Comment (Lctrs) [inline README.md]:
```suggestion
* [composer](https://github.com/composer/composer) (1.10.0)
```

**Pull request merged**